app/styles/components/footer.css

```diff
@@ -0,0 +1,25 @@
+footer {
```

Comment:
You shouldn't need any of this - it should all come from the ember-styleguide ui addon.

Reply:
I looked over for long time and tried to add tht link in `<EsFooter>` tag but i couldn't able to add.
I looked in ember docs about how to add our own content in that footer tag and tried that too but it does not update the default text in that tag. 
So thats why i writed a css separately.
How to add our content in that tag?